examples/angular/src/pages/ui/components/authenticator/sign-in-with-email/sign-in-with-email.component.ts

```diff
@@ -1,10 +1,11 @@
-import { Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
```

Comment:
Ooops, I think this snuck in here

Reply:
Oh lol that explains why CI wasn't passing